application/shared-kernel/ApiCore/Endpoints/TrackEndpoints.cs

```diff
@@ -0,0 +1,206 @@
+using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.DataContracts;
+using Microsoft.ApplicationInsights.Extensibility.Implementation;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Routing;
+using StackFrame = Microsoft.ApplicationInsights.DataContracts.StackFrame;
+
+namespace PlatformPlatform.SharedKernel.ApiCore.Endpoints;
+
+public static class TrackEndpoints
```

Comment:
Talking about documentation, we should probably also describe why we have this endpoint instead of just letting the JavaScript SDK report directly to Application Insights.